Altis_Life.Altis/stringtable.xml

```diff
@@ -4329,7 +4329,7 @@
             <Polish>Jest to nielegalne element i policjanci są w pobliżu. Nie można pozbyć się dowodów.</Polish>
             <Portuguese>Este é um item ilegal e policiais estão nas proximidades. Você não pode dispor das provas.</Portuguese>
             <Russian>Это незаконный пункт и полицейские находятся поблизости. Вы не можете избавиться от доказательств.</Russian>
-            <German>Dies ist eine illegale Artikel und Polizisten sind in der Nähe. Sie können nicht von den Beweisen zu entsorgen.</German>
+            <German>Dies ist eine illegaler Gegenstand und Polizisten sind in der Nähe. Du kannst die Beweise nicht entsorgen.</German>
```

Comment:
`Dies ist ein illegaler Gegenstand und Polizisten sind in der Nähe. Du kannst jetzt die Beweise nicht einfach so entsorgen.`